pandas-stubs/core/indexes/multi.pyi

```diff
@@ -125,36 +159,33 @@ class MultiIndex(Index):
     def __getitem__(  # pyright: ignore[reportIncompatibleMethodOverride] # ty: ignore[invalid-method-override]
         self, key: int
     ) -> tuple[Hashable, ...]: ...
-    def append(self, other): ...
-    def repeat(self, repeats, axis=...): ...
-    def drop(self, codes, level: Level | None = None, errors: str = "raise") -> Self: ...  # type: ignore[override] # pyright: ignore[reportIncompatibleMethodOverride]
+    @overload  # type: ignore[override]
+    def append(self, other: MultiIndex | Sequence[MultiIndex]) -> MultiIndex: ...
+    @overload
+    def append(  # pyright: ignore[reportIncompatibleMethodOverride]
+        self, other: Index | Sequence[Index]
+    ) -> Index: ...  # pyrefly: ignore[bad-override]
+    def drop(self, codes: Level | Sequence[Level], level: Level | None = None, errors: str = "raise") -> MultiIndex: ...  # type: ignore[override] # pyright: ignore[reportIncompatibleMethodOverride]
     def swaplevel(self, i: int = -2, j: int = -1) -> Self: ...
-    def reorder_levels(self, order): ...
+    def reorder_levels(self, order: Sequence[Level]) -> MultiIndex: ...
     def sortlevel(
         self,
         level: Level | Sequence[Level] = 0,
         ascending: bool = True,
         sort_remaining: bool = True,
         na_position: NaPosition = "first",
-    ): ...
-    @final
-    def get_indexer(self, target, method=..., limit=..., tolerance=...): ...
-    def get_indexer_non_unique(self, target): ...
-    def reindex(self, target, method=..., level=..., limit=..., tolerance=...): ...
-    def get_slice_bound(
-        self, label: Hashable | Sequence[Hashable], side: str
-    ) -> int: ...
+    ) -> tuple[MultiIndex, np_1darray_intp]: ...
     def get_loc_level(
-        self, key, level: Level | list[Level] | None = None, drop_level: bool = True
-    ): ...
-    def get_locs(self, seq): ...
+        self,
+        key: Label | Sequence[Label],
+        level: Level | Sequence[Level] | None = None,
+        drop_level: bool = True,
+    ) -> tuple[int | slice | np_1darray_bool, Index]: ...
+    def get_locs(self, seq: Level | Sequence[Level]) -> np_1darray_intp: ...
     def truncate(
         self, before: IndexLabel | None = None, after: IndexLabel | None = None
-    ): ...
-    def equals(self, other) -> bool: ...
-    def equal_levels(self, other): ...
-    def insert(self, loc, item): ...
-    def delete(self, loc): ...
+    ) -> MultiIndex: ...
+    def equal_levels(self, other: MultiIndex) -> bool: ...
```

Comment:
Is this actually documented? I can't find it in the docs

Reply:
Could not find it at all, prob worth deleting unless you have seen use cases in production.